userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md

```diff
@@ -107,6 +109,7 @@ and destination of your PR. For example, the following command will:
 
 ```bash
 pctl add \
+  --name pctl-profile \
```

Comment:
```suggestion
  --name <profile installation name> \
```

Reply:
the example here has dummy names of the flags like `--pr-repo drwho/thirteen` so i am keeping the name suggestion here

Reply:
can we have a more meaningful example then? `nginx-profile` or something like that

Reply:
i think this is fine
```
pctl add \
  --name pctl-profile \
  --profile-repo-url https://github.com/weaveworks/nginx-profile \
  --create-pr \
  --pr-repo drwho/thirteen \
  --out ethel \
  --namespace two-sugars-please \
  --pr-branch add-aunt-ethels-profile \
  --pr-remote jammy-dodger \
  --pr-base tea-time
  ```

Reply:
`pctl-profile` doesn't really mean anything though, its just the name of the two tools :shrug:

Reply:
the rest of the example doesn't have meaningful names either 😂  let's not be pedantic 😄

Reply:
I sort of dislike that the rest aren't useful :joy: (although `--profile-repo-url` is a real example) the docs should help users make better decisions, e.g. good branch/installation namings